misk/src/main/kotlin/misk/tasks/RepeatedTaskQueue.kt

```diff
@@ -100,7 +114,7 @@ class RepeatedTaskQueue @VisibleForTesting internal constructor(
     failureBackoff: Backoff = ExponentialBackoff(timeBetweenRuns, defaultMaxDelay, defaultJitter),
     task: () -> Status
   ) {
-    val wrappedTask: () -> Result = {
+    schedule(delay = initialDelay) {
```

Comment:
this is kind of strange since it results in double wrapping the task. can this just call enqueue?

Reply:
https://github.com/square/misk/pull/960